extensions/BMO/Extension.pm

```diff
@@ -2253,6 +2253,7 @@ sub forced_format {
   # note: this is also called from the guided bug entry extension
   my ($product, $component) = @_;
   return undef unless defined $product;
+  $component ||= '';
```

Comment:
This seems weird, but if this is the conventional way for the default value of a Perl argument to be equivalent to `None`, then :+1:

Reply:
Yeah. For older versions of Perl that is the customary way to give default values. Python is much nicer in this regard. Newer Perl has support for experimental features such as 'signatures' which gives Perl more Python like syntax.

Reply:
https://www.effectiveperlprogramming.com/2015/04/use-v5-20-subroutine-signatures/